packages/core/src/providers/passkey.ts

```diff
@@ -0,0 +1,84 @@
+/**
+ * <div style={{backgroundColor: "#24292f", display: "flex", justifyContent: "space-between", color: "#fff", padding: 16}}>
+ * <span>Built-in <b>Passkey</b> integration.</span>
+ * <a href="https://passkeys.dev">
+ *   <img style={{display: "block"}} src="https://authjs.dev/img/providers/passkey.svg" height="48" width="48"/>
+ * </a>
+ * </div>
+ *
+ * @module providers/passkey
+ */
+
+import WebAuthn, { WebAuthnInputConfig, WebAuthnConfig, DEFAULT_WEBAUTHN_TIMEOUT } from "./webauthn"
+
+
+/**
+ * Add Passkey login to your page.
+ * 
+ * ### Setup
+ * 
+ * #### Configuration
+ * ```ts
+ * import { Auth } from "@auth/core"
+ * import Passkey from "@auth/core/providers/passkey"
+ *
+ * const request = new Request(origin)
+ * const response = await Auth(request, {
+ *   providers: [Passkey({ relayingParty: { id: "example.com", name: "My Website", origin: "https://example.com" } })],
```

Comment:
our current convention is to make this config minimal, I suggest we can infer the relayingParty's fields from env vars prefixed with `AUTH_`, how do you think?